pkg/azuredx/azureauth/auth.go

```diff
@@ -0,0 +1,133 @@
+// Package azureauth provides authorization utilities for the Azure cloud.
+package azureauth
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/grafana/azure-data-explorer-datasource/pkg/azuredx/models"
+)
+
+var onBehalfOfLatencySeconds = prometheus.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
Any idea why this metric won't show up at the Grafana `/metrics` collection @alexanderzobnin?

Reply:
Started a bug report <https://github.com/grafana/grafana-plugin-sdk-go/issues/439> now.